app/assets/javascripts/application.js

```diff
@@ -15,3 +15,9 @@
 //= require turbolinks
 //= require cartify/application
 //= require_tree .
+function openDiv(categories) {
+    let i, tabCategories;
```

Comment:
I can't see that you actually use the variable `i` anymore... Please remove it.

Reply:
we're using it on line 23